golem_core/demand_builder/props.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+from decimal import Decimal
+from typing import Optional
+
+from dataclasses import dataclass
+
+from golem_core.demand_builder.model import Model, prop
+
+
+@dataclass
+class NodeInfo(Model):
+    """Properties describing the information regarding the node."""
+
+    name: Optional[str] = prop("golem.node.id.name", default=None)
+    """human-readable name of the Golem node"""
+
+    subnet_tag: Optional[str] = prop("golem.node.debug.subnet", default=None)
+    """the name of the subnet within which the Demands and Offers are matched"""
```

Comment:
Why `"""..."""` instead of `# ...` and why comment is located under the attribute?

Reply:
Concept found in yapapi, and moved as-is.